import_3dm.py

```diff
@@ -1,3 +1,15 @@
+bl_info = {
+    "name": "Import Rhinoceros 3D",
+    "author": "jesterKing",
+    "version": (0, 0, 1),
+    "blender": (2, 80, 0),
+    "location": "File > Import > Rhinoceros 3D (.3dm)",
+    "description": "This Addon lets you import all render meshes from Rhinoceros 3dm files",
```

Comment:
I would word this a little bit different

```suggestion
    "description": "This addon lets you import Rhinoceros 3dm files",
```

Reply:
I had same text as you suggest in the beginning but the I thought there should be information on the fact that for the time being only rendermeshes are supported. This is of course solved in a much clearer way in the "warning" line as you suggested! :) I applied these changes.